packages/@aws-cdk/aws-amplify/README.md

```diff
@@ -15,8 +15,79 @@
 ---
 <!--END STABILITY BANNER-->
 
-This module is part of the [AWS Cloud Development Kit](https://github.com/aws/aws-cdk) project.
+Define a new App:
+```ts
+new App(this, 'MyApp', {
```

Comment:
use `amplify.App`. This is the idiomatic way we reference resources within construct library modules and also we want to disambiguate with `core.App`, which we idiomatically refer to as `App`.